src/data.js

```diff
@@ -0,0 +1,199 @@
+              <li>We are disappointed by some actors. Projects need better supports. It's our role to help them and build a software craftsmanship organization.</li>
+            </ol>
+          </p>
+        </Fragment>
+    },
+    {
+      badge: "fab fa-github",
+      title: "Default git template and truffle stack",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> Mars 2018
+
+            <p>Alexandre release some tools for our projects and our external audits on "Smart Contracts".</p>
+          </small>
+        </Fragment>
+    },
+    {
+      position : "timeline-inverted",
+      badge: "fab fa-github",
+      title: "First pull request on Bitcoin for Johan",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> Mars 2018
+          </small>
+
+          <p>Johan release his dynamic build tool for wallet and masternode and he made this
+            first pull-request on Bitcoin.</p>
+        </Fragment>
+    },
+    {
+      position : "",
+      badge: "fab fa-github",
+      title: "Release of our old projects",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> February 2018
+          </small>
+
+          <p>Almost all of our personals projects are on Github!</p>
+        </Fragment>
+    },
+    {
+      position : "timeline-centered",
+      color: "primary",
+      badge: "fas fa-microphone",
+      title: "First public talk",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> February 2018
+          </small>
+
+          <p>
+            Our first public talk about "Le Petit Bloc". At this moment, we are just a startup studio for blockchain projects.
+          </p>
+        </Fragment>
+    },
+    {
+      position : "timeline-inverted",
+      color: "warning",
+      badge: "fab fa-github",
+      title: "Creation of our Github account",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> December 2017
+          </small>
+
+          <p>Hello <a href="https://github.com/LePetitBloc" target="_blank">Github</a>!</p>
+        </Fragment>
+    },
+    {
+      position : "timeline-centered",
+      color: "primary",
+      badge: "far fa-comment-dots",
+      title: "Project development",
+      content:
+        <Fragment>
+          <small className="text-muted">
+            <i className="fas fa-calendar-alt"></i> September 2017
+          </small>
+
+          <p>
+            After a year of experiments, Johan and Alexandre started to talk about a their projects about some
+            "somethings" in blockchain. Codename: "dotchain".
```

Comment:
After a year of experimentation, Johan and Alexandre started to talk about a common project about  "something" in the blockchain economy. Codename: "dotchain".